data.js

```diff
@@ -4368,5 +4368,33 @@ module.exports = [
     url: 'https://mahdix.com',
     twitter: '@mahdix',
     tags: ['Java', 'SaaS', 'Kotlin', 'Linux'],
-  }
+  },
+  {
+    name: "Manthan Ank",
+    description: "I'm Front-End Developer, I write tutorials related to Web Development.",
+    url: "https://dev.to/manthanank",
+    twitter: "@manthanank",
+    tags: [
```

Comment:
Hi @manthanank ! Kindly lessen the tags to at least **10** tags only. Feel free to just commit here again in the PR. Lmk if you need any help.

Reply:
removed few tags